examples/reactnative/rn063example/android/app/build.gradle

```diff
@@ -73,11 +78,10 @@ import com.android.build.OutputFile
  */
 
 project.ext.react = [
-    entryFile: "index.js"
+    enableHermes: false,  // clean and rebuild if changing
 ]
 
 apply from: "../../node_modules/react-native/react.gradle"
-apply from: "../../node_modules/@bugsnag/react-native/bugsnag-react-native.gradle"
```

Comment:
this is required to resolve the AAR dependency on Android

Reply:
It seemed to work fine without, but more than happy to add - especially if it's something that can fail intermittently.

Reply:
Ok, it's coming in the next release.  I've added this and bumped the dependency to the next alpha release.  CI will fail initially, but once the alpha is created it should pass on retry.